dist/svg.js

```diff
@@ -1540,10 +1542,27 @@ SVG.FX = SVG.invent({
       return this
     }
 
+    // set the internal animation pointer at the start position and updates the visualisation
+  , atStart: function() {
+    // Cast boolean to number
+    return this.at(+this.situation.reversed)
+  }
+
+    // set the internal animation pointer at the end position and updates the visualisation
+  , atEnd: function() {
+    // Cast boolean to number
+    return this.at(+!this.situation.reversed)
+  }
+
     // set the internal animation pointer to the specified position and updates the visualisation
   , at: function(pos){
       var durDivSpd = this.situation.duration/this._speed
 
+      // correct position
+      if(pos > 1) pos = 1
+      if(pos < 0) pos = 0
+      if(this.situation.reversed) pos = 1 - pos
```

Comment:
hm indeed... do you see a way to let the position go backwards instead of 1-pos? That would simplify things a lot. Preferable this is done in the final `step()` methods so other methods do not have to bother about reversing stuff. Just a question. Because I dont have an idea expect calculating the position completely different when animation is reversed :/